plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/attribute-filter/components/attribute-dropdown.tsx

```diff
@@ -1,25 +1,38 @@
 /**
  * External dependencies
  */
+import styled from '@emotion/styled';
 import FormTokenField from '@woocommerce/base-components/form-token-field';
 import { __, sprintf } from '@wordpress/i18n';
 import { Icon, chevronDown } from '@wordpress/icons';
 
 type Props = {
 	label: string;
+	textColor: string;
+};
+
+export const AttributeDropdown = ( { label, textColor }: Props ) => {
+	const StyledFormTokenField = textColor
+		? styled( FormTokenField )`
+				.components-form-token-field__input::placeholder {
+					color: ${ textColor } !important;
+				}
+		  `
+		: FormTokenField;
```

Comment:
If we can use CSS only here also, we don't need to use styled component, hence reducing lots of logic.